main.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable linebreak-style */
+// /* eslint-disable no-undef
+// eslint-disable no-use-before-define
+/* eslint-disable linebreak-style */
+// create the book constructor
+
+function Book(title, author, isbn) {
+  this.title = title;
+  this.author = author;
+  this.isbn = isbn;
+}
+// UI constructor
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.